src/toil/jobStores/aws/jobStore.py

```diff
@@ -228,6 +272,48 @@ def getEmptyFileStoreID(self, jobStoreID=None):
         log.debug("Created %r.", info)
         return info.fileID
 
+    def _importFile(self, otherCls, url):
+        if issubclass(otherCls, type(self)):
```

Comment:
`if issubclass(otherCls, AWSJobStore):`

because self could be an instance of a sibling or cousin of otherCls.